pulser/simulation/simresults.py

```diff
@@ -104,8 +106,9 @@ def get_final_state(self, reduce_to_basis: Optional[str] = None,
             elif reduce_to_basis == "digital":
                 ex_state = "0"
             else:
-                raise ValueError("'reduce_to_basis' must be 'ground-rydberg' "
-                                 + f"or 'digital', not '{reduce_to_basis}'.")
+                raise ValueError(
+                    "'reduce_to_basis' must be 'ground-rydberg' "
+                    + f"'digital' or 'XY', not '{reduce_to_basis}'.")
```

Comment:
Actually, `reduce_to_basis` is not supported for `XY`, because it's meaningless (you will always have just a two-level system, so there's no reduction to be made). You should state this in the docstring of this method and change this message back.